dask/utils_test.py

```diff
@@ -5,10 +5,18 @@
 import time
 from typing import TYPE_CHECKING
 
+import pandas as pd
```

Comment:
I think similar comment here too. My guess is we use this module in portions of the codebase that don't rely on `pandas` being installed